settings.gradle

```diff
@@ -1,3 +1,4 @@
 rootProject.name = 'concoction'
 include('concoction-lib', 'concoction-cli')
-project(":concoction-lib").name = 'concoction'
\ No newline at end of file
+project(":concoction-lib").name = 'concoction'
+include 'concoction-gui'
```

Comment:
Why is this here? Move this to the `include` above.

Reply:
IntelliJ auto-add moment